### PR TITLE
ADS Welcome Page

### PR DESCRIPTION
**Pull request by chlafreniere**
image

### CODE REVIEW

File: src/vs/workbench/contrib/welcome/page/browser/welcomePage.css [hunk 1]
```diff
@@ -38,6 +38,11 @@
 	overflow: hidden;
 }
 
+/* {{SQL CARBON EDIT}} Overwrite ADS caption display to be initial instead of block */
+.monaco-workbench .part.editor > .content .welcomePage .row .splash .showOnStartup .caption {
```
Comment (anthonydresser) [inline src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: why is this necessary?

Comment (chlafreniere) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: Without this, the checkbox and the "Show welcome page on startup" string are on different lines. Was just trying to scope this down. If you have any other suggestions, more than happy to  entertain them.

Comment (anthonydresser) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: I would be interested in why thats the case for ours but not theirs.

Comment (chlafreniere) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: In sql/base/browser/ui/table/media/table.css we say that every label has display:block, so that's why there's a difference between ADS and VSCode

Comment (anthonydresser) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: We should update that css to be more scoped to table then

Comment (chlafreniere) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: For sure. But changing that could have unintended consequences, especially this late in the release.

Comment (anthonydresser) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: The problem is if this goes into master, the likely hood that we will do the proper fix later is pretty low (based on the past). Maybe a better fix would be scope table in this PR and if you are worried about that affecting release, do a release branch only fix that removes that and adds this?

File: src/vs/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (kburtram) [inline src/vs/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: can we add this file under `sql` directory?  New files should be created in parallel directories under there.

File: src/vs/workbench/contrib/welcome/page/browser/welcomePage.css (hunk 1 above)
Comment (kburtram) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: We can always take a different commit in release/1.6 then in master if we want to make different fixes in each branch.

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		<div class="row">
+			<div class="splash">
+				<div class="section start">
+					<h2 class="caption">${escape(localize('welcomePage.start', "Start"))}</h2>
+					<ul>
+						<li><a href="command:registeredServers.addConnection">${escape(localize('welcomePage.newConnection', "New connection"))}</a></li>
+						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
+						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
+						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
+						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
```
Comment (YurongHe) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: dup?

Comment (YurongHe) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: Are we going to add Open Folder for next release?

Comment (chlafreniere) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: I fixed the 2 links issue; will only see 1.

I worked with @yualan who came up with the list of items to put on the welcome page. Let's ask him 😄

File: src/vs/workbench/contrib/welcome/page/browser/welcomePage.css (hunk 1 above)
Comment (chlafreniere) [reply src/vs/workbench/contrib/welcome/page/browser/welcomePage.css]: I've fixed this. Looked at tables and compared screenshots for results; couldn't find a difference. Removed the edit in welcomePage.css.
